app/src/main/java/ch/epfl/sweng/erpa/operations/RemoteServicesProviderCoordinator.java

```diff
@@ -74,7 +74,7 @@ public void bindRemoteServicesProvider(@Nullable Class<? extends RemoteServicesP
         sharedPreferences.edit().putString(REMOTE_PROVIDER_KEY, rspClassName).apply();
     }
 
-    private Optional<Class<? extends RemoteServicesProvider>> rspClassFromApplicationPreferences() {
+    Optional<Class<? extends RemoteServicesProvider>> rspClassFromApplicationPreferences() {
```

Comment:
Why did you remove the private?

Reply:
I think you did that, at least it was in your tmp commit

Reply:
Check if you can leave it private

Reply:
Nope, can't do